taxonomies/food/categories.txt

```diff
@@ -69663,6 +69669,10 @@ hr: Suhe borovnice
 lt: Džiovintos šilauogės
 nl: Gedroogde bosbessen
 
+< en:Dried fruits
+en: Dried cherries
+fr: Cerises séchées
+
```

Comment:
This is not necessary because dried/sechees are already in ingredients_processing taxonom. Hence, dried cherries and cerises sechees should be recognized ingredients already.